packages/m/megacmd/package.yml

```diff
@@ -0,0 +1,42 @@
+name       : megacmd
+version    : 1.6.3
+release    : 1
+source     :
+    - https://github.com/meganz/MEGAcmd/archive/refs/tags/1.6.3_Linux.tar.gz : d0378d2a6787f72467eadf27c82145f232b456bc7cd8719abb0ce51e7f719796
+    - https://github.com/meganz/sdk/archive/965e4c7cd518d48c5b2fbf0084a244fdecc44b35.tar.gz : a3e1b96f27bdc60a0b46b50e95731a750d2fa18dcd379e0f6d022e62c6dc653e
+homepage   : https://mega.io/cmd
+license    : BSD-2-Clause
+component  : network.download
+summary    : Command Line Interactive and Scriptable Application to access MEGA
+description: |
+    MEGAcmd provides non UI access to MEGA services. It intends to offer all the functionality with your MEGA account via commands. It features synchronization , backup of local folders into your MEGA account and a webdav/streaming server.
+builddeps  :
+    - pkgconfig(libcares)
+    - pkgconfig(libcryptopp)
+    - pkgconfig(libcurl)
+    - pkgconfig(libmediainfo)
+    - pkgconfig(libraw)
+    - pkgconfig(libsodium)
+    - pkgconfig(libuv)
+    - pkgconfig(sqlite3)
+    - ffmpeg-devel
+    - freeimage-devel
+    - libicu-devel
+    - pcre2-devel
```

Comment:
`pkgconfig(libpcre2-posix)`, or one of the other pkgconfig deps that look like they make sense (look at `eopkg info pcre2-devel` to see what it provides).